backstop/tests/tile.html

```diff
@@ -310,8 +310,9 @@ <h2>Default</h2>
     <section id="demo-default">
       <!-- No thumbnail -->
       <div
-        class="iui-tile"
+        class="iui-tile iui-actionable"
```

Comment:
Tile width override in html can be also removed as tile has width set of same value

Reply:
Removed tile width from HTML head.